fragments/sample.json

```diff
@@ -15,6 +15,15 @@
                 "BucketName": {
                     "Ref": "BucketName"
                 },
+                "BucketEncryption": {
+                    "ServerSideEncryptionConfiguration": [
+                        {
+                            "ServerSideEncryptionByDefault": {
+                                "SSEAlgorithm": "AES256"
+                            }
+                        }
+                    ]
+                },
```

Comment:
This whole file really doesn't need to be here. It's an artifact of the modules unit tests. I wasn't able to figure out how to get rid of it today with my time constraints, but will do this in a separate PR.